R/geom-jitter.r

```diff
@@ -39,7 +39,7 @@ geom_jitter <- function(mapping = NULL, data = NULL,
                         inherit.aes = TRUE) {
   if (!missing(width) || !missing(height)) {
     if (!missing(position)) {
-      stop("Specify either `position` or `width`/`height`", call. = FALSE)
+      stop("You must specify one of `position` and `width`/`height`", call. = FALSE)
```

Comment:
Proposed mashup of old/new:
"You must specify either `position` or `width`/`height`"?

Reply:
Would it be terrible to add "but not both"? The reason why I started this conversation in the first place was that I felt the error message didn't sufficiently clearly express that the error is that both options are specified at once.

Reply:
I think the `either...or` construct provides that.

Reply:
Ok.

@yutannihilation Would you mind updating the error text to what @batpigandme proposed, and then I'll merge?